### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by pbanaszkiewicz** [swcarpentry/amy#384]
After a very little bit of thinking I selected some areas where we need improvements:

1. We're missing `aria` HTML attributes
2. We're missing elements for screen readers only
3. …and elements that should be shown only in non-screen reader browsers
4. We might need to change focus order for elements like links and forms

This is not a high priority.

There are many articles about accessibility. Bootstrap docs and https://medium.com/salesforce-ux/7-things-every-designer-needs-to-know-about-accessibility-64f105f0881b for the start are good.
